### DISCUSSION TITLE
Toggling pvp outside claim

### DISCUSSION THREAD

**Discussion by MithrandirCraft** [GriefPrevention/GriefPrevention#790] [Ideas]
Add **/griefprevention pvp <true/false>**, and or corresponding configuration option. (preferably at very least, the command)

**Is your feature request related to a problem? Please describe.**
If I'm not mistaken, this plugin's design simply depends on world gamerules, or worldguard flags, for pvp to be either enabled or disabled outside of claims (and even inside if the world isn't included in the pvp rules) The issue with this is that there's simply no way to keep the roads between claims peaceful unless, pvp is locked globally in the world.

I was planning on adding extra flags, so if I block pvp globally using gamerules or worldguard, I then can't allow players to make per claim pvp available.

The solution would be an extra option for determining if pvp is allowed outside claims.

**Describe alternatives you've considered**
WorldGuard/Vanilla gamerules. They are not feasable for the mentioned reasons.

**Comment by RoboMWM** [comment]
Yea, GriefPrevention flags plugin may be helpful. I've also made my own plugin/addon for PvP. https://github.com/TechFortress/DisablePvP

**Comment by MithrandirCraft** [comment]
I just noticed grief prevention extra flags may do this. Just verify and close the ticket if so.

**Comment by MithrandirCraft** [comment]
![imagen](https://user-images.githubusercontent.com/6623704/80052100-22efb480-851a-11ea-8e74-cb7837e2e959.png)
Actually GPExtraFlags can't do this. So I'll have to take a look at your other addon.

**Comment by RoboMWM** [comment]
GriefPrevention Flags, maintained by @ShaneBeee is a fork of the addon Big_Scary started for GP.

**Comment by MithrandirCraft** [comment]
Feel free to close if you do not wish to implement this feature. 
Just please manage pvp better in v20.
